clippy_dev/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(tool_lints)]
+#![allow(clippy::default_hash_types)]
```

Comment:
@phansch in `clippy_dev` the feature `rustc_private` isn't activated, because no `use rustc::...` is used. That means that  also `FxHashMap` cannot be used.

I think disabling the Lint here is better than adding the `#![feature(rustc_private)]`. What's your opinion on this?

Reply:
Yea we can just disable the lint there. I don't think we will need any `rustc::...` in `clippy_dev` in the near future.